notebooks/04_model_select_and_optimize/azureml_hyperdrive_wide_and_deep.ipynb

```diff
@@ -7,7 +7,7 @@
     "<i>Copyright (c) Microsoft Corporation. All rights reserved.<br>\n",
```

Comment:
<p>also what changed here to impact the metrics? are we not setting a random seed when training the model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
The w&d notebook does use the default seed unless we explicitly set `None`.
Will test again and see if it reports the same number again (should be).